spec.html

```diff
@@ -41524,9 +41559,9 @@ <h1>
           1. Assert: _W_ is equivalent to AgentSignifier().
           1. Assert: _W_ is on the list of waiters in _WL_.
           1. Assert: AgentCanSuspend() is *true*.
-          1. Perform LeaveCriticalSection(_WL_) and suspend _W_ for up to _timeout_ milliseconds, performing the combined operation in such a way that a notification that arrives after the critical section is exited but before the suspension takes effect is not lost. _W_ can notify either because the timeout expired or because it was notified explicitly by another agent calling NotifyWaiter(_WL_, _W_), and not for any other reasons at all.
+          1. Perform LeaveCriticalSection(_WL_) and suspend _W_ for up to _timeout_ milliseconds, performing the combined operation in such a way that a notification that arrives after the critical section is exited but before the suspension takes effect is not lost. _W_ can notify either because the timeout expired or because it was notified explicitly by another agent calling NotifyWaiter with arguments _WL_ and _W_, and not for any other reasons at all.
           1. Perform EnterCriticalSection(_WL_).
-          1. If _W_ was notified explicitly by another agent calling NotifyWaiter(_WL_, _W_), return *true*.
+          1. If _W_ was notified explicitly by another agent calling NotifyWaiter with arguments _WL_ and _W_, return *true*.
```

Comment:
Not sure why these were changed to SDO-style "with arguments". NotifyWaiter is invoked with a parenthesized list of args, so why wouldn't these references to hypothetical invocations be formatted that way too?

Reply:
Two reasons:

1. We didn't want these references to be treated as calls by ecmarkup because (if they were) they're not used appropriately for the types.
2. There's actually ambiguity in the previous phrasing. Are we referring to the result of the call? This makes it clear that we are referring to `NotifyWaiter` itself.

Reply:
Okay, make sense.

Reply:
@ljharb This is another little thing that can be pulled into a separate PR if you like.

Reply:
It would need to land first, though.

Reply:
#2667